packages/logger/package.json

```diff
@@ -28,12 +28,12 @@
   "scripts": {
     "prepare": "npm run build",
     "build": "npm run build:clean && tsc",
-    "build:clean": "shx rm -rf ./dist ./coverage ./.nyc_output",
+    "build:clean": "shx rm -rf ./dist ./coverage",
     "lint": "eslint --ext .ts src",
     "mocha": "mocha --config .mocharc.json src/*.spec.js",
     "test:unit": "npm run build && npm run mocha",
-    "test": "npm run lint && npm run test:unit",
-    "coverage": "npm run build && nyc --reporter=text-summary npm run mocha",
+    "test": "npm run lint && npm run coverage",
```

Comment:
~Should this run `test:unit` before `coverage`?~

Edit: Ah `c8 npm run mocha` has the test I'm looking for!